addon-test-support/@ember/test-helpers/dom/-get-element.ts

```diff
@@ -1,7 +1,13 @@
 import getRootElement from './get-root-element';
 import Target, { isDocument, isElement } from './-target';
 
-function getElement(target: string): Element | null;
+function getElement<K extends keyof HTMLElementTagNameMap>(
+  target: K
+): HTMLElementTagNameMap[K] | null;
+function getElement<K extends keyof SVGElementTagNameMap>(
+  target: K
+): SVGElementTagNameMap[K] | null;
+function getElement<E extends Element = Element>(target: string): E | null;
```

Comment:
I recognize these came from the DT types, but… this is terrible and we should not perpetuate its terribleness.

```suggestion
export default function getElement(selector: string): Element | null;
```

This change prevents people from “casting” it implicitly by writing `getElement<HTMLDivElement>()`. Although that's *convenient*, especially in tests, it's also quite misleading.